data/json/monsters/zed_misc.json

```
@@ -211,6 +213,7 @@
     "weight": "200 kg",
     "hp": 300,
     "speed": 240,
+    "attack_cost": 240,
```

The description specifically calls out that it's overall very fast, not just at movement.
```suggestion
    "attack_cost": 120,
```